internal/settings/settings.go

```diff
@@ -9,13 +9,22 @@ import (
 	"os"
 	"path"
 
-	"github.com/OpenPeeDeeP/xdg"
 	"github.com/elastic/elastic-agent-changelog-tool/internal/gitreporoot"
 	"github.com/spf13/viper"
 )
 
-const envPrefix = "ELASTIC_AGENT_CHANGELOG"
-const configFileFolder = "elastic-agent-changelog-tool"
+const (
+	defaultOwner = "elastic"
+	defaultRepo  = "elastic-agent"
```

Comment:
```suggestion
```

I would remove these from here and leverage flag default value. These values are a duplicate and would not be updated properly, so I don't think it make sense setting them in viper.